src/components/parameters/Icons.svelte.test.ts

```diff
@@ -0,0 +1,16 @@
+import ActivityIcon from '@nasa-jpl/stellar/icons/svg/plan.svg?component';
```

Comment:
Can we move this test to the general `components/ui` directory?

Reply:
Ah yep whoops thanks for catching that.